File: BEE 1010.cpp
#include<iostream>

#include<iomanip>
#define FIXED_FLOAT(x) std::fixed << std::setprecision(2) << (x)

using namespace std;

int main(){
    int p1, p2, u1, u2;
    float price1, price2;
    cin>>p1>>u1>>price1;
    cin>>p2>>u2>>price2;
    cout<<"VALOR A PAGAR: R$ "<<FIXED_FLOAT((u1*price1)+(u2*price2))<<endl;
    return 0;
}

File: BEE 1010.java
import java.io.IOException;
import java.util.Scanner;

public class Main{
    public static void main(String[] args) throws IOException{
        Scanner input = new Scanner(System.in);
        int p1, p2, u1, u2;
        float price1, price2;
        p1 = input.nextInt();
        u1 = input.nextInt();
        price1 = input.nextFloat();
        p2 = input.nextInt();
        u2 = input.nextInt();
        price2 = input.nextFloat();
        System.out.printf("VALOR A PAGAR: R$ %.2f\n", ((u1*price1)+(u2*price2)));
    }
}